diff --git a/src/librpbase/config/Config.cpp b/src/librpbase/config/Config.cpp
index 048ba9629..ae72f63c8 100644
--- a/src/librpbase/config/Config.cpp
+++ b/src/librpbase/config/Config.cpp
@@ -103,7 +103,7 @@ public:
 
 	// Download options
 	static const constexpr uint32_t palLanguageForGameTDB_default = 'en';
-	static const constexpr bool extImgDownloadEnabled_default = true;
+	static const constexpr bool extImgDownloadEnabled_default = false;
 	static const constexpr bool useIntIconForSmallSizes_default = true;
 	static const constexpr bool storeFileOriginInfo_default = true;
 
